core/src/main/java/de/danoeh/antennapod/core/feed/SubscriptionFilter.java

```diff
@@ -0,0 +1,127 @@
+package de.danoeh.antennapod.core.feed;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class SubscriptionFilter {
+    public static final String STRING_ENABLE = "Enable";
+    public static final String STRING_DISABLE = "Disable";
+    public static final String STRING_ALWAYS = "Always";
+    public static final String STRING_NEVER = "Never";
+    public static final String STRING_GLOBAL_DEFAULT = "Global default";
+    public static final String STRING_NO_FILTER = "No filter";
+
+    public enum AutoDownload{
+        ENABLE(STRING_ENABLE),
+        DISABLE(STRING_DISABLE),
+        NO_FILTER(STRING_NO_FILTER);
+        private String autoDownloadString;
+        AutoDownload(String autoDownloadStringValue) {
+            autoDownloadString = autoDownloadStringValue;
+        }
+
+        @Override
+        public String toString(){
+            return autoDownloadString;
+        }
+
+        public static AutoDownload parse(String autoDownloadValue){
+            switch (autoDownloadValue){
+                case STRING_ENABLE: return AutoDownload.ENABLE;
+                case STRING_DISABLE: return AutoDownload.DISABLE;
+                case STRING_NO_FILTER: default: return AutoDownload.NO_FILTER;
+            }
+        }
+    }
+    public enum KeepUpdated{
+        ENABLE(STRING_ENABLE),
+        DISABLE(STRING_DISABLE),
+        NO_FILTER(STRING_NO_FILTER);
+        private String keepUpdatedString;
+        KeepUpdated(String keepUpdatedStringValue) {
+            keepUpdatedString = keepUpdatedStringValue;
+        }
+
+        @Override
+        public String toString(){
```

Comment:
Please add a space before opening curly braces. This should be done everywhere in the code you touch (but not in the whole project. It is inconsistent there, too, but I want it to slowly get more consistent).

```suggestion
        public String toString() {
```